python/pylibraft/pyproject.toml

```diff
@@ -19,7 +19,7 @@ requires = [
     "cuda-python>=11.7.1,<12.0",
     "cython>=0.29,<0.30",
     "ninja",
-    "rmm==23.8.*",
+    "rmm==23.6.*",
```

Comment:
Why is this required?

Reply:
@bdice Yes, I think the `dependencies.yaml` still has 23.6, so the right fix is to update that to 23.8

Reply:
Fix is filed here: https://github.com/rapidsai/raft/pull/1556